Link_sinFade_Library.ino

```diff
@@ -260,62 +263,63 @@ void HSVehtoRGB(float HSVfColor[], float rgbf_color[]) {
   float Sat = HSVfColor[1];  // 0 to 100.
   float Val = HSVfColor[2];  // 0 to 100.
 
-  int rangeNumber = HSVehRange(HSVfColor);      ////////  Uses: HSVehRange(hsvColorF).  <<<< Can I do away with "rangeNumber" ?
+  // rgb's are in 0.0 ... 1.0
+  int rangeNumber = HSVehRange(HSVfColor);      ////////  Uses: HSVehRange(hsvColorF).  <<<< FIXME: Can I do away with "rangeNumber" ?
   switch (rangeNumber) {
     case 1:    // Range 1 - HuNum = 0 - 31 ... RGB - {255, xx, 0}   (Maj, Mid, Min)
-      rgbf_color[0] = Val * 2.55;                             // Maj
-      rgbf_color[1] = Val * (255 - Sat * (255 - 2.845 * Hue) / 100) / 100; // Mid
-      rgbf_color[2] = Val * 2.55 * (100 - Sat) / 100;         // Min
+      rgbf_color[0] = Val;                             // Maj
+      rgbf_color[1] = Val * (255 - Sat * (255 - 2.845 * Hue) / 100) / 100 / 255; // Mid (punt on 255 thing)
```

Comment:
same pattern change all the way down. I did test my change to the expressions to make sure they gave the same results.